pkgs/tools/networking/edgedb/default.nix

```
@@ -17,16 +17,16 @@
 
 rustPlatform.buildRustPackage rec {
   pname = "edgedb";
-  version = "unstable-2022-06-27";
+  version = "1.2.1";
 
   src = fetchFromGitHub {
     owner = "edgedb";
     repo = "edgedb-cli";
-    rev = "3c65c8bf0a09988356ad477d0ae234182f809b0a";
-    sha256 = "sha256-UqoRa5ZbPJEHo9wyyygrN1ssorgY3cLw/mMrCDXr4gw=";
+    rev = "v{version}";
```

You missed the dollar sign.
```suggestion
    rev = "v${version}";
```

---

:sweat_smile: Sorry, got distracted and forgot to mess with a hash to trigger a proper rebuild.